src/drive/web/modules/public/LightFileList.jsx

```diff
@@ -23,7 +24,9 @@ const LightFileList = ({
   </>
 )
 
-export default connect(
-  mapStateToProps,
-  mapDispatchToProps
-)(LightFileList)
+export default withRouter(
+  connect(
+    mapStateToProps,
+    mapDispatchToProps
+  )(LightFileList)
+)
```

Comment:
I don't understand this change. It seems we don't use `router` in this component and we don't pass it neither to its children. Can you explain why this is needed?

Reply:
It is used [here](https://github.com/cozy/cozy-drive/blame/fix-public/src/drive/web/modules/public/Actions.jsx#L54-L55), in the `mapDispatchToProps` of he component. But I agree, it's not very obvious. Do you think it would be better to include the `Actions` file in this one to make it more explicit? Otherwise we can add a comment imo.

Reply:
The best solution is to not rely on `ownprops.something` (since it can be dangerous for the perf)  but I don't know how to handle that.